src/aave-v2/SupplyVaultBase.sol

```diff
@@ -76,6 +79,10 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
 
     /// EXTERNAL ///
 
+    /// @dev Vaults are not intended to hold ERC20 tokens, so this only owner function transfers a specified asset to a recipient.
+    /// @param _asset The asset to rescue.
```

Comment:
Should we keep "to rescue"?

Reply:
Should we even comment these parameters, as the comments don't bring any new information to the reader (beyond the description implied by their name)?

Otherwise `rescue` could be replaced by `transfer`

Reply:
This comment should be changed anyways if we change the scope of this function

Reply:
I agree with changing it, but since we are changing the scope of the function I will just remove these added comments for now.

Reply:
Yep agree!